sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -196,6 +196,22 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
             raise ValueError("No token scope provided.")
         return _generate_sas_token(scopes[0], self.policy, self.key)
 
+class AzureNamedKeyTokenCredential(object):
```

Comment:
same comment as in the [eventhub PR](https://github.com/Azure/azure-sdk-for-python/pull/18292), rename to something like:
```suggestion
class ServiceBusAzureNamedKeyCredential(object):
```